index.js

```diff
@@ -0,0 +1,14 @@
+"use strict";
+
+alert("First number is larger than Second number");
+
+let firstNumber = prompt("Enter first number");
+let secondNumber = prompt("Enter second number");
+
+let compare;
+
+if (firstNumber > secondNumber) {
+    alert(compare = "I am bigger");
```

Comment:
Having alert(compare="I am bigger"), is a bit confusing.
Would be more readable if you did separately do value assigning and afterwards alert.
Also message "I am bigger" is not very explaining. There will be questions "Who is bigger?","Bigger than what?"
Try to create more informative messages, and code in general, because other developers will be working with your code as well. 👍